webapp/components/at_mention/at_mention.jsx

```diff
@@ -79,13 +80,15 @@ export default class AtMention extends React.PureComponent {
                     placement='right'
                     rootClose={true}
                     overlay={
-                        <ProfilePopover
-                            user={user}
-                            src={Client4.getProfilePictureUrl(user.id, user.last_picture_update)}
-                            hide={this.hideProfilePopover}
-                            isRHS={this.props.isRHS}
-                            hasMention={this.props.hasMention}
-                        />
+                        <Pluggable>
+                            <ProfilePopover
+                                user={user}
+                                src={Client4.getProfilePictureUrl(user.id, user.last_picture_update)}
+                                hide={this.hideProfilePopover}
+                                isRHS={this.props.isRHS}
+                                hasMention={this.props.hasMention}
+                            />
+                        </Pluggable>
```

Comment:
It's possible to create a new component for `ProfilePopover`, maybe `ProfilePopoverPluggable` so that can be imported instead of explicitly wrapping the component everywhere